libs/inngest/src/integrations/website/types.ts

```diff
@@ -16,6 +16,17 @@ export const WebDevChallengeFormSubmit = z.object({
 	links: z.array(z.object({ label: z.string(), url: z.string() })),
 });
 
+export const WebDevChallengeHackathonSubmit = z.object({
+	id: z.string(),
+	username: z.string(),
+	email: z.string(),
+	fullName: z.string(),
+	githubRepo: z.string(),
+	deployedApp: z.string(),
+	tocAgreement: z.boolean(),
+	doNotShare: z.boolean(),
```

Comment:
TODO: check if this one needs to be optional

Reply:
The `doNotShare` should be optional. I can make another PR to fix that later.